### DISCUSSION TITLE
Starcoin, Aptos, Sui 使用的Move语言是否互通？差别大吗？

### DISCUSSION THREAD

**Discussion by Congyuwang** [NonceGeek/Move-Camp#151] [Q&A] [answered]
Starcoin, Aptos, Sui都使用move语言进行开发。但是听闻aptos和sui都是对该语言有一定修订的，那么他们的差别大吗？

**Comment by DaviRain-Su** [comment, marked as answer]
starcoin 和aptos之间应该是没有区别的，都是集成了move官方，都是使用全局存储，但是sui的move 对数据的存储是有自己的sui object存储。https://zhuanlan.zhihu.com/p/563459834 这个是sui的move和官方move之间的不同比较
